CHANGELOG.rst

```diff
@@ -7,6 +7,15 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 1.5.1 (2023-05-03)
+--------------------------
+
+* Changed: require ``audeer>=1.20.0``
+* Fixed: a bunch of ``audb`` functions
```

Comment:
I think it was only `audb.load()` and `audb.remove_media()`.

Reply:
Also `audb.load_to()` (or if that depends directly on `audb.load()` then nvm)

Reply:
Maybe also `load_media()` and `audb.dependencies()`. I think it is easier to not try to find out which functions were effected.

But maybe change to `some` instead of  `a bunch of`?

Reply:
Maybe before release, we can include this https://github.com/audeering/audb/pull/304 ?

Reply:
I checked and were able to track down the following functions:
* `audb.load()`
* `audb.load_to()`
* `audb.remove_media()`

I updated the changelog accordingly.